samples/config_helper_vulkan.cc

```diff
@@ -667,8 +667,7 @@ amber::Result ConfigHelperVulkan::CreateVulkanInstance(
     uint32_t engine_major,
     uint32_t engine_minor,
     std::vector<std::string> required_extensions,
-    bool disable_validation_layer,
-    bool show_version_info) {
```

Comment:
Why was the `show_version_info` flag removed?

Reply:
The flag was used in this function to decide whether or not require `VK_KHR_get_physical_device_properties2` extension. The first patch changes the function to always require that extension if available.